include/rocksdb/env.h

```diff
@@ -475,7 +482,8 @@ class Env {
   virtual void LowerThreadPoolIOPriority(Priority /*pool*/ = LOW) {}
 
   // Lower CPU priority for threads from the specified pool.
-  virtual void LowerThreadPoolCPUPriority(Priority /*pool*/ = LOW) {}
+  virtual void LowerThreadPoolCPUPriority(
+      Priority /*pool*/ = LOW, CpuPriority /*pri*/ = CpuPriority::kLow) {}
```

Comment:
In my opinion, we should avoid default argument for both.

Since it is a new function, can we add a return value, and return not supported by default? Maybe it can be used for others in the future.

Reply:
I see `Env::LowerThreadPoolCPUPriority()` already exists before this change.

Reply:
existing users might have already called `LowerThreadPoolCPUPriority()`, with these default values, the method keeps the original semantics.

Reply:
To clarify, I mean to remove default value for both parameters in this new function. What's the point of having a default parameter value which doesn't serve any compatibility purpose?

Reply:
Can you double check it is new? It does not look new to me.

Reply:
OK I think I finally understood. `LowerThreadPoolCPUPriority(Priority /*pool*/ = LOW, CpuPriority /*pri*/ = CpuPriority::kLow)` *should* be a new function because deleting `virtual void LowerThreadPoolCPUPriority(Priority /*pool*/ = LOW)` would break compatibility for existing `Env` implementations. I agree with that. The actionable change here though is first to reintroduce `virtual void LowerThreadPoolCPUPriority(Priority /*pool*/ = LOW)`, and then make the two-argument overload have non-default arguments.

Reply:
considering compatibility with Env implementations out of our control, this makes sense to me now.